src/setup-beam.js

```diff
@@ -321,6 +346,14 @@ function isStrictVersion() {
   return getInput('version-type', false) === 'strict'
 }
 
+function isExactVersion(spec) {
+  if (semver.parse(spec)) {
```

Comment:
@paulo-ferraz-oliveira  Is there a way to have the linter ignore this function? It's complaining because of useless else, which is somewhat true, but also I don't trust javascript when it comes to boolean logic and `undefined` 😁

Reply:
Ignore this. I went ahead and made a "this is fine" version 😉